server/tests-py/conftest.py

```diff
@@ -39,6 +39,10 @@ def pytest_addoption(parser):
         "--hge-jwt-conf", metavar="HGE_JWT_CONF", help="The JWT conf", required=False
     )
 
+    parser.addoption(
+        "--hge-jwt-claims-ns-path", metavar="HGE_JWT_CLAIMS_NS_PATH", help="The JWT conf claims namespace path", required=False
```

Comment:
Isn't `claims_namespace_path` present in `--hge-jwt-conf`?

Reply:
Yes, I have modified the logic to parse the `claims_namespace_path` from `hge-jwt-conf`